client/store/examples/activity.js

```diff
@@ -17,24 +20,55 @@ export const mutations = {
     state,
     values
   ) {
+    /**
+     * We are not dealing with id columns here.
+     * That should be done in some way.
+     * Because, as it is, we may end up with two
+     * entries with the same id.
+     */
     for (const activity of values) {
-      state.activities.push(Object.assign({}, activity))
+      state.activities.push({...activity})
```

Comment:
This will need refactoring.
I'm unsure the recommended way with Element-UI and table rows.
My gut feeling tells me it's better to have an object in the Vuex store, with an unique identifier as a key, so we can iterate.
Yet give an array when asked for the entries.

Because as this is, at any dispatch, the store will grow. With duplicates and all.

What's your feeling about that @clarkdo ?

Reply:
This is refered to as **The examples are not updated, but always appended with duplicates** in **BUGS** in PR detail